tools/cassandra/cqlclient.go

```diff
@@ -130,6 +128,25 @@ func newCQLClient(cfg *CQLClientConfig) (*cqlClient, error) {
 	return cqlClient, nil
 }
 
+func (cfg *CQLClientConfig) toCassandraConfig() *config.Cassandra {
+	cassandraConfig := config.Cassandra{
+		Hosts:    cfg.Hosts,
+		Port:     cfg.Port,
+		User:     cfg.User,
+		Password: cfg.Password,
+		Keyspace: cfg.Keyspace,
+	}
+
+	if cfg.UseTLS {
+		cassandraConfig.TLS = &config.TLS{
```

Comment:
you want to pass through all of the cassandra options - including validations and CA, not just TLSCertPath / TLSKeyPath